rpm/a11y-sound-theme/a11y.spec

```diff
@@ -0,0 +1,34 @@
+Name:     a11y-sound-theme
+Version:  1
+Release:  2
+Summary:  Linux-A11y sound themes
+License:  GPL-3.0
+
+BuildRequires: git
+
+SOURCE1:  12-panel-a11y
```

Comment:
Hello @kk-at-redhat  and thank you for changes. Actually I have realized one thing - I think we should not force configuration of the sound theme on the user. They should be able to decide if they want to set it as default on their own. At least, that's what I would expect.
Previously this configuration was part of the kickstart because the sound theme was kinda integral part of that kickstart as well... but now when we decided to ship it separately, I think we do not want this.
What do you think?

Reply:
I think it comes down to what would be the most comfortable for our target audience, and I'm hardly the right person to give an input on that. If you think it's better not to enable it by default I can go ahead and remove the config part.
From logical point IMHO if we announce a11y is included in our ISO, i.e. it's one of the features, then we might as well enable it and add something in the documentation that explains how to change that.
So just let me know what you want to do.

Reply:
I think we can do it in a way that everyone is satisfied. I suggest we drop the dconf configuration from the package, but we can keep it in the kickstart file.
My thinking is this: This package can be installed by anyone who subscribes to our copr repo. If I install the package on already installed system, I probably don't expect that it will automatically change my sound theme.
On the other hand, when building Vojtux ISO image, it is fine to have this theme set as default.
Do you agree? Could you please do those changes?

Reply:
AFter gathering fedback from the community and after discussion with @kk-at-redhat  I take back my suggestion of removing the dconf configuration from this package.